src/main/java/co/aurasphere/botmill/fb/autoreply/Reply.java

```diff
@@ -26,11 +26,12 @@
 import co.aurasphere.botmill.fb.model.incoming.MessageEnvelope;
 import co.aurasphere.botmill.fb.model.outcoming.FbBotMillResponse;
 
+
 /**
  * Interface which represents every reply from the bot.
  * 
  * @author Donato Rimenti
- * @since 1.2.0
+ * @author Alvin Reyes
  */
```

Comment:
I was using the since annotation to keep track of what we added and when. Is there a reason why you removed it?